docs/commands.md

```diff
@@ -7,81 +9,92 @@
 
 ## Push
 
-Push allows users to create a bundle from files or directories on their local file systems and
-then push the resulting artifact to a registry. For example,
+### Overview
 
-`$ imgpkg push -f my-bundle -b index.docker.io/k8slt/sample-bundle`
+`push` command allows users to create a bundle in the registry from files and/or directories on their local file systems. For example,
 
-will push a bundle containing the `my-bundle` directory to `index.docker.io/k8slt/sample-bundle`.
+```bash
+$ imgpkg push -b index.docker.io/k8slt/sample-bundle -f my-bundle/
+```
 
-The `-f` flag can be used multiple times to add different files or directories to the bundle.
+will push a bundle contents containing in the `my-bundle/` directory to `index.docker.io/k8slt/sample-bundle`.
 
-Use the `-b`/`--bundle` flag to specify the destination of the push.
-If the specified destination does not include a tag, the artifact will be pushed with the default tag `:latest`.
+Use the `-b`/`--bundle` flag to specify the destination of the push. If the specified destination does not include a tag, the artifact will be pushed with the default tag `:latest`.
+
+The `-f` flag can be used multiple times to add different files or directories to the bundle.
 
 ### Generating a [BundleLock](resources.md#bundlelock)
 
-`push` may also output a BundleLock file for users that would like a deterministic reference to a pushed bundle. For example, running:
+`push` command can output a [`BundleLock` configuration](resources.md#bundlelock-configuration) for users that would like a deterministic reference to a pushed bundle. For example, running:
 
-`$ impgpkg push -f my-bundle -b index.docker.io/k8slt/sample-bundle:v0.1.0 --lock-output
-bundle.lock.yml`
+```bash
+$ impgpkg push -b index.docker.io/k8slt/sample-bundle:v0.1.0 -f my-bundle/ --lock-output
+/tmp/bundle.lock.yml
+```
+
+will create `/tmp/bundle.lock.yml` with BundleLock configuration. If another bundle image in the repository is later given the same tag (`v0.1.0`), the BundleLock configuration will continue to provide immutable reference (via digest) to the original pushed bundle.
```

Comment:
```suggestion
will create a BundleLock configuration in `/tmp/bundle.lock.yml`. If another bundle image in the repository is later given the same tag (`v0.1.0`), the BundleLock configuration will continue to provide immutable reference (via digest) to the original pushed bundle.
```

Reply:
we are creating a file that contains BundleLock configuration.